icons/npm/npm-plain.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><rect fill="#cc3534" x="4.83" y="4.83" width="118.35" height="118.35" rx="6.53" ry="6.53" /><path fill="#fff" d="M27.934 101.526h36.141l.046-54.119h18.024l-.046 54.165h18.07l.047-72.189-72.19-.092-.092 72.235z" /></svg>
```

Comment:
This file should be renamed to original. It's multi-colored so it does not fit the criteria for plain versions, you can read more about the requirements and guidelines for each version here: https://github.com/devicons/devicon/wiki/SVG-Versions

Reply:
Thanks for your review, this is my first one :)

I renamed the file and updated and the JSON. I also added a new plain version.
Let me know if i made this right.

Reply:
I resized the icons and changed the color.